test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -1934,14 +1934,21 @@ public void BinaryClassifierLDSvmNoCalibTest()
         /// <summary>
         /// A test for field-aware factorization machine.
         /// </summary>
-        [Fact(Skip = "Need CoreTLC specific baseline update")]
+        [Fact]
         [TestCategory("Binary")]
         [TestCategory("FieldAwareFactorizationMachine")]
         public void BinaryClassifierFieldAwareFactorizationMachineTest()
         {
             var binaryPredictors = new[] { TestLearners.FieldAwareFactorizationMachine };
             var binaryClassificationDatasets = GetDatasetsForBinaryClassifierBaseTest();
-            RunAllTests(binaryPredictors, binaryClassificationDatasets);
+
+            // see https://github.com/dotnet/machinelearning/issues/404
+            // in Linux, the clang sqrt() results vary highly from the ones in mac and Windows. 
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                RunAllTests(binaryPredictors, binaryClassificationDatasets, digitsOfPrecision:4);
```

Comment:
Does it mean that we only check "9487" in "9487.05"? The issue you opened said that the difference happens at the 17th decimal, so probably we can increate it to 7?

Reply:
In the small subset that I explored, there were differences starting on the 4th decimal digit, for Linux. Should update the issue. 

---
In reply to: [223888003](https://github.com/dotnet/machinelearning/pull/1206#discussion_r223888003) [](ancestors = 223888003)

Reply:
giving 5 another try after updating the comparison code. 

---
In reply to: [224267254](https://github.com/dotnet/machinelearning/pull/1206#discussion_r224267254) [](ancestors = 224267254,223888003)